catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -1004,34 +1002,34 @@ public CreateResponse create(CreateRequest createRequest)
                 }
                 metacard.setAttribute(new AttributeImpl(Metacard.SECURITY, itemPolicyMap));
             }
-            createReq.getProperties()
+            createRequest.getProperties()
                     .put(PolicyPlugin.OPERATION_SECURITY, requestPolicyMap);
 
             for (AccessPlugin plugin : frameworkProperties.getAccessPlugins()) {
-                createReq = plugin.processPreCreate(createReq);
+                createRequest = plugin.processPreCreate(createRequest);
             }
 
-            createReq.getProperties()
+            createRequest.getProperties()
                     .put(Constants.OPERATION_TRANSACTION_KEY,
                             new OperationTransactionImpl(OperationTransaction.OperationType.CREATE,
                                     new ArrayList<>()));
 
             for (PreIngestPlugin plugin : frameworkProperties.getPreIngest()) {
                 try {
-                    createReq = plugin.process(createReq);
+                    createRequest = plugin.process(createRequest);
                 } catch (PluginExecutionException e) {
                     LOGGER.info(
                             "Plugin processing failed. This is allowable. Skipping to next plugin.",
                             e);
                 }
             }
-            validateCreateRequest(createReq);
+            validateCreateRequest(createRequest);
 
             // Call the create on the catalog
-            LOGGER.debug("Calling catalog.create() with {} entries.", createReq.getMetacards()
+            LOGGER.debug("Calling catalog.create() with {} entries.", createRequest.getMetacards()
                     .size());
 
-            if (Requests.isLocal(createReq)) {
+            if (Requests.isLocal(createRequest)) {
                 createResponse = catalog.create(createRequest);
```

Comment:
Actually, it's being used in a couple different try blocks, so as it's written it's fine but using the ternary operator won't work since it would be in only one of the try blocks.